src/math/FGFunction.cpp

```diff
@@ -527,6 +527,15 @@ void FGFunction::Load(Element* el, FGPropertyValue* var, FGFDMExec* fdmex,
                  return y != 0.0 ? fmod(p[0]->GetValue(), y) : HUGE_VAL;
                };
       Parameters.push_back(new aFunc<decltype(f), 2>(f, fdmex, element, Prefix, var));
+    } else if (operation == "roundmultiple") {
+      auto f = [](const decltype(Parameters)& p)->double {
+                 double multiple = p.size() == 1 ? 1.0 : p[1]->GetValue();
+                 return round((p[0]->GetValue() / multiple)) * multiple;
+               };
+      if (element->GetNumElements() == 1)
+        Parameters.push_back(new aFunc<decltype(f), 1>(f, fdmex, element, Prefix, var));
+      else
+        Parameters.push_back(new aFunc<decltype(f), 2>(f, fdmex, element, Prefix, var));
```

Comment:
Fix the function declaration so that it reports an error if the number of arguments is either 0 or higher than 2. With the current code in the PR, the error message will tell that exactly 2 arguments must be supplied which is not what we want since `<roundmultiply>` also accepts a single argument.
```suggestion
        Parameters.push_back(new aFunc<decltype(f), 1>(f, fdmex, element, Prefix, var, 2));
```
Indeed if `<roundmultiply>` is supplied with only one argument, this branch will not be executed. If there are no arguments or more than 2 arguments, this branch will be executed and will report an error. Therefore, this branch will only succeed if 2 arguments are supplied to `<roundmultiply>`. Hence the proposed modification.